protobuf/emif/adapter/schema/submission_status.proto

```diff
@@ -42,9 +42,6 @@ message SubmissionStatus {
     
     // Represents a colleciton of metrics regarding a submission
     message Metrics {
-        // The time when the payload was enqueued
-        google.protobuf.Timestamp enqueue_time = 1;
-
         // The time when the payload was read of the queue
         google.protobuf.Timestamp dequeue_time = 2;
```

Comment:
Burde påfølgende felter få oppdatert feltnummer her, eller må det være slik for bakoverkompatibilitet?
```suggestion
        google.protobuf.Timestamp dequeue_time = 1;
```